assignments/callbacks.js

```diff
@@ -31,4 +46,14 @@ function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+  const arr2 = array.slice();
+  console.log(arr2);
+  let res = [];
+  for (i = 0; i < arr2.length; i++){
+    if (res.includes(arr2[i]) != true){
```

Comment:
I strech you to try using the includes that you build but it would work similarly. :+1: